docs/source/conf.py

```diff
@@ -58,8 +59,7 @@
 templates_path = ["_templates"]
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "zenburn"
```

Comment:
why change this?

Reply:
let's keep the current pygments styles

Reply:
it makes the background looks better on light theme